pkg/system/azure_utils.go

```diff
@@ -43,6 +43,7 @@ func (r *Reconciler) CreateStorageAccount(accountName, accountGroupName string)
 	// if the name is not available, CreateStorageAccount will return an error, and a different name will be used next time
 
 	enableHTTPSTrafficOnly := true
+	AllowBlobPublicAccess := false
```

Comment:
just to keep with conventions, start with lower case
```suggestion
	allowBlobPublicAccess := false
```

Reply:
done